config/coredns/Corefile

```diff
@@ -1,11 +1,15 @@
 k.example.com {
     debug
     errors
+    log
+    
+    rewrite name regex kuadrant-active-groups\.(.*)k.example\.com kuadrant-active-groups-coredns.pb.hcpapps.net
+    forward kuadrant-active-groups-coredns.pb.hcpapps.net /etc/resolv.conf
```

Comment:
this will need to be updated to allow this custom host to be passed in at set up time, somehow, that's for another ticket: https://github.com/Kuadrant/dns-operator/issues/670